Tests/BugsnagTests/ReporterTests.swift

```diff
@@ -277,55 +240,23 @@ class ReporterTests: XCTestCase {
 
     }
 
-    func testErrorBeingReportedWhenNilReleaseStages() {
```

Comment:
This test is still relevant I guess? This is the case I was referring to when `notifyReleaseStages` are not set (=nil).